CONTRIBUTING.md

```diff
@@ -258,13 +258,13 @@ All traces modules defined:
 
 Other methods used by some trace modules:
 
-- `_module.categories`: list of string identifies used to grouped traces by behavior
+- `_module.categories`: list of string identifiers used to group traces by behavior
```

Comment:
Can you say something about what these categories are used for, and how to figure out what categories to use for your new trace?